index.js

```diff
@@ -66,7 +65,26 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+// const changeToggle = document.querySelector('h1#header')
+// changeToggle.addEventListener('click', function () {
+//     document.body.classList.toggle('dark-mode')
+// })
+const changeToggle = document.querySelector('h1#header')
```

Comment:
Remember that we have the header variable already defined earlier in the file.

- All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

- The callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.

```javascript
header.addEventListener("click", function(e){
    toggleColor(e.target)
})
```
With ES6 notation
```javascript
header.addEventListener("click", e => toggleColor(e.target))
```